scripts/cibuild

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n ${RASTER_VISION_DEBUG} ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Build library for integration or a release.
+"
+}
+
+if [[ ${BASH_SOURCE[0]} == "$0" ]]; then
+    if [[ ${1:-} == "--help" ]]; then
+        usage
+    else
+        docker build \
+            --build-arg CUDA_VERSION="10.2" \
+            -t "raster-vision-${IMAGE_TYPE}" \
+            -f Dockerfile \
+            .
```

Comment:
I've generally removed conditional checks for whether we're building a pytorch image and interpolated in the `IMAGE_TYPE` instead. If we introduce other variant types later on and need to parameterize other things like the Cuda version, we can add it to the build matrix.